### DISCUSSION TITLE
Dogecoin Core 1.14.7

### DISCUSSION THREAD

**Discussion by patricklodder** [dogecoin/dogecoin#3452] [General]
This is a new minor version release, including enhancements to several RPC methods and important security updates for Dogecoin-Qt. 

***Dogecoin-Qt users on any platform are strongly recommended to upgrade.***

A full description of all changes is available in the [release notes](https://github.com/dogecoin/dogecoin/blob/master/doc/release-notes.md)

### Security Updates

- Disable BIP-70 payment server by default
- Reduce the build scope of the Qt dependency
- Backport patches for all Qt versions until 5.15.12 + community patches to the pinned Qt 5.7.1
- Added security guidance to INSTALL.md
- Updated the `rpcuser.py` script to improve the password and salt generation methods it uses

### Enhancements

- `estimatefee` and `estimatesmartfee` are now maintained and tuned to Dogecoin parametrization
- Added fully verbose transaction decoding to `getblock`
- Added a `height` parameter to `-walletnotify`
- Added the `getblockstats` RPC method

### RPC Changes

- `verbosity` parameter to `getblock` now accepts both integers and booleans
- Added a `height` parameter to `importpubkey` and `importaddress`

### Translation Updates

- Chinese
- Dutch
- French
- French-Canadian
- Italian
- Korean
- Polish

### Updates to pinned dependencies and subtrees

- Updated the depends, CI and build system to Ubuntu Focal
- Updated expat to version 2.5.0
- Updated the secp256k1 subtree to `44c2452`
- Updated zlib to version 1.3
- Updated protobuf to version 3.6.1
- Updated fontconfig to version 2.12.6
- Removed dependency on libx11

<hr /><em>This discussion was created from the release <a href='https://github.com/dogecoin/dogecoin/releases/tag/v1.14.7'>Dogecoin Core 1.14.7</a>.</em>

**Comment by mikatomic** [comment]
Just downloaded. Thanks for all the hard work you and the team have put into this update and all the others, Patrick!

**Comment by Mantrine** [comment]
Thanks you very much for the work. One question. Why don't arm & aarch64 versions have "dogecoin-qt" GUI interface?

**Reply by patricklodder** [reply]
> Why don't arm & aarch64 versions have "dogecoin-qt" GUI interface?

There have been issues making reproducible builds for those platforms while retaining backward compatibility. With the upcoming major release we don't have these constraints, making this much easier to realize. So let's see if we can get that done - for aarch64 to start with.

**Reply by Mantrine** [reply]
Oukei. Great to hear : D

**Comment by raziyehbah41** [comment]
چرا برای ios ندارد که نصب کنیم

**Comment by georgeartem** [comment]
Could you kindly translate the last comment into English for us stupid and lazy Americans please?Sent from my iPhoneOn Apr 3, 2024, at 8:07 AM, raziyehbah41 ***@***.***> wrote:﻿
چرا برای ios ندارد که نصب کنیم

**Reply by raziyehbah41** [reply]
I have an account on [dogechine.info](http://dogechine.info/) and I need 2FA to enter the account, and when I enter the id wallet, no email containing 2FA is sent to my gmail. What should I do? I am very upset.

**Reply by Ekene1998** [reply]
Text me now on telegram @Lylar0702

**Comment by raziyehbah41** [comment]
I have an account on dogechine.info and I need 2FA to enter the account,
and when I enter the id wallet, no email containing 2FA is sent to my
gmail. What should I do? I am very upset.

**Comment by twdurdentwd** [comment]
WIN64 showing Malware.
![Screenshot 2024-07-25 225354](https://github.com/user-attachments/assets/9e4ca590-2314-4c78-9397-2de4abab3ae1)

**Reply by patricklodder** [reply]
Could you please share the details?

**Reply by twdurdentwd** [reply]
See the attachment, please
